libtcmu.c

```diff
@@ -544,34 +544,37 @@ static void remove_device(struct tcmulib_context *ctx,
 static int is_uio(const struct dirent *dirent)
 {
 	int fd;
-	char tmp_path[64];
-	char buf[256];
-	ssize_t ret;
+	char *tmp_path;
+	char buf[256] = {'\0'};
```

Comment:
Was the compiler warning about buf? What arguments did you use to see the warnings?